app/components/action-feed/component.js

```diff
@@ -12,15 +12,57 @@ import { translationMacro as t } from 'ember-i18n';
  * ```
  * @class action-feed
  */
+
 export default Component.extend({
     store: service(),
     toast: service(),
-
+    currentUser: service(),
     classNames: ['action-feed'],
+    page: 1,
+    listLoading: false,
 
     errorMessage: t('components.action-feed.error_loading'),
 
     moreActions: computed('totalPages', 'page', function() {
         return this.get('page') < this.get('totalPages');
     }),
+
+    init() {
+        this._super(...arguments);
+        this.loadPage();
```

Comment:
Minor: It probably wouldn't ever cause a problem, since the promises will take time to resolve, but it seems wrong to call `loadPage` before initializing the other properties.